### PR TITLE
chore: change deploy driver's name

### PR DESCRIPTION
**Pull request by Siglud**

### CODE REVIEW

**Review by yukun-dong — APPROVED** [review]

**Review by hund030 — DISMISSED** [review]

**Review by swatDong — DISMISSED** [review]

File: adaptive-card-notification/teamsapp.yml [hunk 1]
```diff
@@ -44,12 +44,13 @@ deploy:
...
+  # Deploy your application to Azure Functions using the zip deploy feature.
+  # For additional details, please refer to https://aka.ms/zip-deploy-to-azure-functions
+  - uses: azureFunctions/zipDeploy
     with:
-      # deploy base folder
-      distributionPath: ./
+      artifactFolder: .
       # can be changed to any ignore file location, leave blank will ignore nothing
-      ignoreFile: ./.funcignore
+      ignoreFile: .funcignore
```
Comment (eriolchan) [inline adaptive-card-notification/teamsapp.yml:53]: I have one question that why "ignoreFile: ./.webappignore" and "ignoreFile: .funcignore" here?

File: graph-toolkit-contact-exporter/teamsapp.yml [hunk 2]
```diff
@@ -74,10 +74,13 @@ deploy:
...
     with:
       args: run build
-  - uses: azureStorage/deploy # Deploy bits to Azure Storage Static Website
+  # Deploy bits to Azure Storage Static Website
+  - uses: azureStorage/deploy
     with:
-      distributionPath: ./build # Deploy base folder. This folder includes manifest files for AAD app and Teams app that should be ignored using the ignoreFile.
-      resourceId: ${{TAB_AZURE_STORAGE_RESOURCE_ID}} # The resource id of the cloud resource to be deployed to. This key will be generated by arm/deploy action automatically. You can replace it with your existing Azure Resource id or add it to your environment variable file.
+      # Deploy base folder. This folder includes manifest files for AAD app and Teams app that should be ignored using the ignoreFile.
+      artifactFolder: ./build
```
Comment (eriolchan) [inline graph-toolkit-contact-exporter/teamsapp.yml:81]: Same here, why we use "artifactFolder: build" in graph-connector-app/teamsapp.yml, but "artifactFolder: ./build" here?

File: adaptive-card-notification/teamsapp.yml (hunk 1 above)
Comment (Siglud) [reply adaptive-card-notification/teamsapp.yml:53]: I don't believe there is a strict rule for this naming pattern, as both options are correct. However, I do think that the old pattern may have too many dots.

**Review by hund030 — APPROVED** [review]

File: query-org-user-with-message-extension-sso/teamsapp.yml [hunk 3]
```diff
@@ -66,14 +66,19 @@ deploy:
...
       REACT_APP_CLIENT_ID: ${{AAD_APP_CLIENT_ID}}
       REACT_APP_START_LOGIN_PAGE_URL: ${{TAB_ENDPOINT}}/auth-start.html
     with:
-      distributionPath: . # Deploy base folder
-      ignoreFile: ./.webappignore # Can be changed to any ignore file location, leave blank will ignore nothing
-      resourceId: ${{BOT_AZURE_APP_SERVICE_RESOURCE_ID}} # The resource id of the cloud resource to be deployed to
+      # Deploy base folder
+      artifactFolder: .
+      # Can be changed to any ignore file location, leave blank will ignore nothing
+      ignoreFile: ./.webappignore
```
Comment (eriolchan) [inline query-org-user-with-message-extension-sso/teamsapp.yml:79]: do we change to ".webappignore"?

File: hello-world-bot-with-tab/teamsapp.yml [hunk 4]
```diff
@@ -59,15 +59,23 @@ deploy:
...
+  # For additional details, please refer to https://aka.ms/zip-deploy-to-app-services.
+  - uses: azureAppService/zipDeploy
     with:
-      distributionPath: ./bot # Deploy base folder
-      ignoreFile: ./bot/.webappignore # Can be changed to any ignore file location, leave blank will ignore nothing
-      resourceId: ${{BOT_AZURE_APP_SERVICE_RESOURCE_ID}} # The resource id of the cloud resource to be deployed to. This key will be generated by arm/deploy action automatically. You can replace it with your existing Azure Resource id or add it to your environment variable file.
+      # Deploy base folder
+      artifactFolder: bot
+      # Can be changed to any ignore file location, leave blank will ignore nothing
+      ignoreFile: ./bot/.webappignore
```
Comment (eriolchan) [inline hello-world-bot-with-tab/teamsapp.yml:76]: Same here

File: command-bot-with-sso/teamsapp.yml [hunk 5]
```diff
@@ -64,11 +64,16 @@ deploy:
...
+  # For additional details, please refer to https://aka.ms/zip-deploy-to-app-services.
+  - uses: azureAppService/zipDeploy
     with:
-      distributionPath: . # Deploy base folder
-      ignoreFile: ./.webappignore # Can be changed to any ignore file location, leave blank will ignore nothing
-      resourceId: ${{BOT_AZURE_APP_SERVICE_RESOURCE_ID}} # The resource id of the cloud resource to be deployed to
+      # Deploy base folder
+      artifactFolder: .
+      # Can be changed to any ignore file location, leave blank will ignore nothing
+      ignoreFile: ./.webappignore
```
Comment (eriolchan) [inline command-bot-with-sso/teamsapp.yml:74]: same here

File: bot-sso/teamsapp.yml [hunk 6]
```diff
@@ -66,11 +66,16 @@ deploy:
...
+  # For additional details, please refer to https://aka.ms/zip-deploy-to-app-services.
+  - uses: azureAppService/zipDeploy
     with:
-      distributionPath: . # Deploy base folder
-      ignoreFile: ./.webappignore # Can be changed to any ignore file location, leave blank will ignore nothing
-      resourceId: ${{BOT_AZURE_APP_SERVICE_RESOURCE_ID}} # The resource id of the cloud resource to be deployed to
+      # Deploy base folder
+      artifactFolder: .
+      # Can be changed to any ignore file location, leave blank will ignore nothing
+      ignoreFile: ./.webappignore
```
Comment (eriolchan) [inline bot-sso/teamsapp.yml:76]: same here

File: NPM-search-connector-M365/teamsapp.yml [hunk 7]
```diff
@@ -55,11 +55,16 @@ deploy:
...
+  # For additional details, please refer to https://aka.ms/zip-deploy-to-app-services.
+  - uses: azureAppService/zipDeploy
     with:
-      distributionPath: . # Deploy base folder
-      ignoreFile: ./.webappignore # Can be changed to any ignore file location, leave blank will ignore nothing
-      resourceId: ${{BOT_AZURE_APP_SERVICE_RESOURCE_ID}} # The resource id of the cloud resource to be deployed to
+      # Deploy base folder
+      artifactFolder: .
+      # Can be changed to any ignore file location, leave blank will ignore nothing
+      ignoreFile: ./.webappignore
```
Comment (eriolchan) [inline NPM-search-connector-M365/teamsapp.yml:65]: same here

**Review by IvanJobs — APPROVED** [review]
lgtm

**Pull request merged**